spec/models/automate_workspace_spec.rb

```diff
@@ -1,7 +1,18 @@
 describe AutomateWorkspace do
   describe "#merge_output!" do
     let(:user) { FactoryGirl.create(:user_with_group, :userid => "admin") }
-    let(:aw) { FactoryGirl.create(:automate_workspace, :user => user, :tenant => user.current_tenant) }
+    let(:aw) do
+      FactoryGirl.create(:automate_workspace, :user   => user,
+                                              :tenant => user.current_tenant,
+                                              :input  => input)
+    end
+    let(:password) { "secret" }
```

Comment:
Those some bonus numbers and special characters in there... popular password is `ca$hcOw` ;)